### ISSUE TITLE
np19 seen as greater than np110 and np111?

### ISSUE THREAD

**Issue by ethankruse** [conda/conda#2407]
I've searched around and see that this has been a recurring issue (e.g. #2359), but this seems to be a different case. When I run conda update --all, it tries to 'update' some packages to the numpy 1.9 build, and downgrade everything else to numpy 1.9

```
The following packages will be UPDATED:
basemap:      1.0.7-np110py27_0  --> 1.0.7-np19py27_0 
iopro:        1.7.2-np110py27_p0 --> 1.7.2-np19py27_p0
numba:        0.25.0-np110py27_0 --> 0.25.0-np19py27_0
scipy:        0.17.0-np111py27_0 --> 0.17.0-np19py27_0
statsmodels:  0.6.1-np111py27_0  --> 0.6.1-np19py27_0 

The following packages will be DOWNGRADED:

apptools:     4.3.0-py27_0       --> 4.2.1-py27_0     
astropy:      1.1.2-np111py27_0  --> 1.0.4-np19py27_0 
enable:       4.5.1-np110py27_0  --> 4.4.1-np19py27_0 
freetype:     2.5.5-0            --> 2.4.10-1         
geos:         3.4.2-0            --> 3.3.3-0          
gevent:       1.1.1-py27_0       --> 1.1.0-py27_0     
glueviz:      0.7.2-py27_0       --> 0.7.1-py27_0     
h5py:         2.6.0-np111py27_1  --> 2.5.0-np19py27_3 
hdf5:         1.8.16-0           --> 1.8.15.1-2       
ipaddress:    1.0.16-py27_0      --> 1.0.14-py27_0    
libpng:       1.6.17-0           --> 1.5.13-1         
matplotlib:   1.5.1-np111py27_0  --> 1.4.3-np19py27_0 
numexpr:      2.5.2-np111py27_0  --> 2.4.4-np19py27_0 
numpy:        1.11.0-py27_0      --> 1.9.3-py27_1     
pandas:       0.18.0-np111py27_0 --> 0.16.2-np19py27_0
pillow:       3.2.0-py27_0       --> 2.7.0-py27_0     
pyface:       4.5.2-py27_0       --> 4.4.0-py27_0     
pyparsing:    2.1.1-py27_0       --> 2.0.3-py27_0     
pyqt:         4.11.4-py27_1      --> 4.10.4-py27_0    
pytables:     3.2.2-np111py27_3  --> 3.2.2-np19py27_0 
qt:           4.8.7-1            --> 4.8.5-3          
queuelib:     1.4.2-py27_0       --> 1.2.2-py27_0     
scikit-image: 0.12.3-np111py27_0 --> 0.11.3-np19py27_0
scikit-learn: 0.17.1-np111py27_0 --> 0.16.1-np19py27_0
traits:       4.5.0-py27_0       --> 4.4.0-py27_0     
traitsui:     4.5.1-py27_0       --> 4.4.0-py27_0     
w3lib:        1.14.2-py27_0      --> 1.12.0-py27_0 
```

But when I run a conda list, nothing seems to depend on numpy 1.9

```
_license                  1.1                      py27_1    defaults
adsbibdesk                3.2.0                     <pip>
alabaster                 0.7.7                    py27_0    defaults
anaconda-client           1.4.0                    py27_0    defaults
appnope                   0.1.0                    py27_0    defaults
apptools                  4.3.0                    py27_0    defaults
argcomplete               1.0.0                    py27_1    defaults
astropy                   1.1.2               np111py27_0    defaults
atom                      0.3.10                   py27_0    defaults
babel                     2.3.3                    py27_0    defaults
backports                 1.0                      py27_0    defaults
backports.shutil-get-terminal-size 1.0.0                     <pip>
backports.ssl-match-hostname 3.5.0.1                   <pip>
backports_abc             0.4                      py27_0    defaults
basemap                   1.0.7               np110py27_0    defaults
bcolz                     1.0.0                    py27_0    defaults
beautifulsoup4            4.4.1                    py27_0    defaults
binstar                   0.12                          2    defaults
bitarray                  0.8.1                    py27_0    <unknown>
blaze                     0.9.1                    py27_0    defaults
blaze-core                0.9.0                    py27_0    defaults
bokeh                     0.11.1                   py27_0    defaults
boto                      2.39.0                   py27_0    defaults
casuarius                 1.1                      py27_0    <unknown>
cdecimal                  2.3                      py27_0    <unknown>
cffi                      1.5.2                    py27_0    defaults
chest                     0.2.3                    py27_0    defaults
cloudpickle               0.2.1                    py27_0    defaults
clyent                    1.2.2                    py27_0    defaults
colorama                  0.3.7                    py27_0    defaults
conda                     4.0.5                    py27_0    defaults
conda-build               1.20.1                   py27_0    defaults
conda-env                 2.4.5                    py27_0    defaults
conda-server              0.14.0                        1    defaults
configobj                 5.0.6                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/configobj-5.0.6-py27_0.tar.bz2
configparser              3.5.0b2                  py27_1    defaults
cryptography              1.3.1                    py27_0    defaults
cssselect                 0.9.1                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/cssselect-0.9.1-py27_0.tar.bz2
cubehelix                 0.1.0                     <pip>
cubes                     1.0.1                    py27_0    defaults
curl                      7.45.0                        0    defaults
cycler                    0.10.0                   py27_0    defaults
cython                    0.24                     py27_0    defaults
cytoolz                   0.7.5                    py27_0    defaults
d2to1                     0.2.12.post1              <pip>
dask                      0.8.2                    py27_0    defaults
datashape                 0.5.1                    py27_0    defaults
dateutil                  2.4.1                    py27_0    defaults
decorator                 4.0.9                    py27_0    defaults
dill                      0.2.5                    py27_0    defaults
distribute                0.6.45                   py27_1    http://repo.continuum.io/pkgs/free/osx-64/distribute-0.6.45-py27_1.tar.bz2
docutils                  0.12                     py27_0    http://repo.continuum.io/pkgs/free/osx-64/docutils-0.12-py27_0.tar.bz2
ecdsa                     0.13                     py27_0    http://repo.continuum.io/pkgs/free/osx-64/ecdsa-0.13-py27_0.tar.bz2
emcee                     2.1.0                     <pip>
enable                    4.5.1               np110py27_0    defaults
enaml                     0.9.8                    py27_1    defaults
entrypoints               0.2                      py27_1    defaults
enum34                    1.1.3                    py27_0    defaults
envisage                  4.4.0                    py27_1    <unknown>
et_xmlfile                1.0.1                    py27_0    defaults
fastcache                 1.0.2                    py27_0    defaults
filelock                  2.0.5                     <pip>
flask                     0.10.1                   py27_1    <unknown>
flask-cors                2.1.2                    py27_0    defaults
freetype                  2.5.5                         0    defaults
funcsigs                  1.0.0                    py27_0    defaults
future                    0.15.2                   py27_0    defaults
futures                   3.0.5                    py27_0    defaults
geos                      3.4.2                         0    defaults
get_terminal_size         1.0.0                    py27_0    defaults
gevent                    1.1.1                    py27_0    defaults
gevent-websocket          0.9.5                    py27_1    defaults
glueviz                   0.7.2                    py27_0    defaults
greenlet                  0.4.9                    py27_0    defaults
grin                      1.2.1                    py27_1    <unknown>
h5py                      2.6.0               np111py27_1    defaults
hdf5                      1.8.16                        0    defaults
heapdict                  1.0.0                    py27_0    defaults
idna                      2.1                      py27_0    defaults
imagesize                 0.7.0                    py27_0    defaults
imaging                   1.1.7                    py27_2    <unknown>
iopro                     1.7.2              np110py27_p0    defaults
ipaddress                 1.0.16                   py27_0    defaults
ipykernel                 4.3.1                    py27_0    defaults
ipython                   4.2.0                    py27_0    defaults
ipython-notebook          4.0.4                    py27_0    defaults
ipython-qtconsole         4.0.1                    py27_0    defaults
ipython_genutils          0.1.0                    py27_0    defaults
ipywidgets                4.1.1                    py27_0    defaults
itsdangerous              0.24                     py27_0    http://repo.continuum.io/pkgs/free/osx-64/itsdangerous-0.24-py27_0.tar.bz2
jbig                      2.1                           0    defaults
jdcal                     1.2                      py27_0    defaults
jedi                      0.9.0                    py27_0    defaults
jinja2                    2.8                      py27_0    defaults
jpeg                      8d                            1    <unknown>
jsonschema                2.4.0                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/jsonschema-2.4.0-py27_0.tar.bz2
jupyter                   1.0.0                    py27_2    defaults
jupyter_client            4.2.2                    py27_0    defaults
jupyter_console           4.1.1                    py27_0    defaults
jupyter_core              4.1.0                    py27_0    defaults
keyring                   3.8                      py27_0    http://repo.continuum.io/pkgs/free/osx-64/keyring-3.8-py27_0.tar.bz2
kiwisolver                0.1.3                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/kiwisolver-0.1.3-py27_0.tar.bz2
krb5                      1.13.2                        0    defaults
launcher                  1.0.0                         3    defaults
lcms                      1.19                          0    http://repo.continuum.io/pkgs/free/osx-64/lcms-1.19-0.tar.bz2
libdynd                   0.7.2                         0    defaults
libevent                  2.0.21                        0    http://repo.continuum.io/pkgs/free/osx-64/libevent-2.0.21-0.tar.bz2
libpng                    1.6.17                        0    defaults
libsodium                 1.0.3                         0    defaults
libtiff                   4.0.6                         0    defaults
libxml2                   2.9.2                         0    defaults
libxslt                   1.1.28                        2    <unknown>
llvm                      3.3                           0    <unknown>
llvmlite                  0.10.0                   py27_0    defaults
llvmpy                    0.12.7                   py27_0    http://repo.continuum.io/pkgs/free/osx-64/llvmpy-0.12.7-py27_0.tar.bz2
locket                    0.2.0                    py27_0    defaults
lxml                      3.6.0                    py27_0    defaults
markupsafe                0.23                     py27_0    http://repo.continuum.io/pkgs/free/osx-64/markupsafe-0.23-py27_0.tar.bz2
matplotlib                1.5.1               np111py27_0    defaults
mdp                       3.5                      py27_0    defaults
meta                      0.4.2.dev                py27_0    <unknown>
mistune                   0.7.2                    py27_1    defaults
mkl                       11.3.1                        0    defaults
mkl-rt                    11.1                         p0    <unknown>
mock                      2.0.0                    py27_0    defaults
mpmath                    0.19                     py27_0    defaults
multipledispatch          0.4.8                    py27_0    defaults
mysql-connector-python    2.0.3                    py27_0    defaults
mysql-python              1.2.5                     <pip>
nbconvert                 4.2.0                    py27_0    defaults
nbformat                  4.0.1                    py27_0    defaults
networkx                  1.11                     py27_0    defaults
nltk                      3.2.1                    py27_0    defaults
node-webkit               0.10.1                        0    http://repo.continuum.io/pkgs/free/osx-64/node-webkit-0.10.1-0.tar.bz2
nose                      1.3.7                    py27_0    defaults
notebook                  4.2.0                    py27_0    defaults
numba                     0.25.0              np110py27_0    defaults
numexpr                   2.5.2               np111py27_0    defaults
numpy                     1.11.0                   py27_0    defaults
odo                       0.4.2                    py27_0    defaults
openpyxl                  2.3.2                    py27_0    defaults
openssl                   1.0.2g                        0    defaults
pandas                    0.18.0              np111py27_0    defaults
paramiko                  1.16.0                   py27_0    defaults
partd                     0.3.2                    py27_1    defaults
path.py                   8.2.1                    py27_0    defaults
patsy                     0.4.1                    py27_0    defaults
pbr                       1.8.0                    py27_0    defaults
pep8                      1.7.0                    py27_0    defaults
pexpect                   4.0.1                    py27_0    defaults
pickleshare               0.5                      py27_0    defaults
pillow                    3.2.0                    py27_0    defaults
pip                       8.1.1                    py27_1    defaults
ply                       3.8                      py27_0    defaults
psutil                    4.1.0                    py27_1    defaults
ptyprocess                0.5                      py27_0    defaults
py                        1.4.31                   py27_0    defaults
pyasn1                    0.1.9                    py27_0    defaults
pyaudio                   0.2.7                    py27_0    <unknown>
pycosat                   0.6.1                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/pycosat-0.6.1-py27_0.tar.bz2
pycparser                 2.14                     py27_0    defaults
pycrypto                  2.6.1                    py27_0    <unknown>
pycurl                    7.19.5.3                 py27_0    defaults
pyface                    4.5.2                    py27_0    defaults
pyflakes                  1.1.0                    py27_0    defaults
pygments                  2.1.3                    py27_0    defaults
pymongo                   3.0.3                    py27_0    defaults
pyopenssl                 0.15.1                   py27_2    defaults
pyparsing                 2.1.1                    py27_0    defaults
pyqt                      4.11.4                   py27_1    defaults
pyraf                     2.1.10                    <pip>
pysal                     1.11.1                   py27_0    defaults
pysam                     0.6                      py27_0    <unknown>
pytables                  3.2.2               np111py27_3    defaults
pytest                    2.9.1                    py27_0    defaults
python                    2.7.11                        0    defaults
python-dateutil           2.5.2                    py27_0    defaults
python.app                1.2                      py27_4    defaults
pytz                      2016.3                   py27_0    defaults
pyyaml                    3.11                     py27_1    defaults
pyzmq                     15.2.0                   py27_0    defaults
qt                        4.8.7                         1    defaults
qtconsole                 4.2.1                    py27_0    defaults
queuelib                  1.4.2                    py27_0    defaults
readline                  6.2                           2    <unknown>
redis                     2.6.9                         0    <unknown>
redis-py                  2.10.3                   py27_0    http://repo.continuum.io/pkgs/free/osx-64/redis-py-2.10.3-py27_0.tar.bz2
requests                  2.9.1                    py27_0    defaults
rope                      0.9.4                    py27_1    http://repo.continuum.io/pkgs/free/osx-64/rope-0.9.4-py27_1.tar.bz2
runipy                    0.1.3                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/runipy-0.1.3-py27_0.tar.bz2
scikit-image              0.12.3              np111py27_0    defaults
scikit-learn              0.17.1              np111py27_0    defaults
scipy                     0.17.0              np111py27_0    defaults
scrapy                    1.0.3                    py27_0    defaults
setuptools                20.7.0                   py27_0    defaults
simplegeneric             0.8.1                    py27_0    defaults
singledispatch            3.4.0.3                  py27_0    defaults
sip                       4.16.9                   py27_0    defaults
six                       1.10.0                   py27_0    defaults
snowballstemmer           1.2.1                    py27_0    defaults
sockjs-tornado            1.0.1                    py27_0    http://repo.continuum.io/pkgs/free/osx-64/sockjs-tornado-1.0.1-py27_0.tar.bz2
sphinx                    1.4.1                    py27_0    defaults
sphinx_rtd_theme          0.1.9                    py27_0    defaults
spyder                    2.3.8                    py27_1    defaults
spyder-app                2.3.8                    py27_0    defaults
sqlalchemy                1.0.12                   py27_0    defaults
sqlite                    3.9.2                         0    defaults
ssl_match_hostname        3.4.0.2                  py27_1    defaults
statsmodels               0.6.1               np111py27_0    defaults
stsci.distutils           0.3.7                     <pip>
stsci.tools               3.4                       <pip>
sympy                     1.0                      py27_0    defaults
tables                    3.2.2                     <pip>
terminado                 0.5                      py27_1    defaults
tk                        8.5.18                        0    defaults
toolz                     0.7.4                    py27_0    defaults
tornado                   4.3                      py27_0    defaults
traitlets                 4.2.1                    py27_0    defaults
traits                    4.5.0                    py27_0    defaults
traitsui                  4.5.1                    py27_0    defaults
twisted                   16.1.1                   py27_0    defaults
ujson                     1.35                     py27_0    defaults
unicodecsv                0.14.1                   py27_0    defaults
vtk                       6.3.0                    py27_1    defaults
w3lib                     1.14.2                   py27_0    defaults
werkzeug                  0.11.8                   py27_0    defaults
wheel                     0.29.0                   py27_0    defaults
xlrd                      0.9.4                    py27_0    defaults
xlsxwriter                0.8.5                    py27_0    defaults
xlwt                      1.0.0                    py27_0    defaults
xz                        5.2.2                         0    defaults
yaml                      0.1.6                         0    defaults
zeromq                    4.1.3                         0    defaults
zlib                      1.2.8                         0    http://repo.continuum.io/pkgs/free/osx-64/zlib-1.2.8-0.tar.bz2
zope.interface            4.1.3                    py27_0    defaults
```

And when I try to figure out what packages I have that are dependent on numpy, none of them list 1.9, so why should conda be trying to downgrade me to numpy 1.9 unless one of those 'upgrades' to the 1.9 build is taking precedence? I also checked and as far as I can tell none of the pip packages require specific versions of numpy, so that shouldn't be an issue. I'm just confused.

Here's the output when I run the recommended `grep numpy ~/anaconda/pkgs/*/info/index.json`:
(I also have my python 3 environment, and those packages are shown too, but you can ignore those I assume.)

```
/Users/ekruse/anaconda/pkgs/abstract-rendering-0.5.1-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/abstract-rendering-0.5.1-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/astropy-1.1.2-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/astropy-1.1.2-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/basemap-1.0.7-np110py27_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/basemap-1.0.7-np110py27_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/basemap-1.0.7-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/basemap-1.0.7-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/bcolz-1.0.0-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/bcolz-1.0.0-py35_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/blaze-core-0.9.0-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/bokeh-0.11.1-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/bokeh-0.11.1-py35_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/dask-0.8.2-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/datashape-0.5.1-py27_0/info/index.json:    "numpy >=1.7", 
/Users/ekruse/anaconda/pkgs/datashape-0.5.1-py35_0/info/index.json:    "numpy >=1.7", 
/Users/ekruse/anaconda/pkgs/enable-4.5.1-np110py27_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/enable-4.5.1-np110py27_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/h5py-2.6.0-np110py35_1/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/h5py-2.6.0-np110py35_1/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/h5py-2.6.0-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/h5py-2.6.0-np111py27_0/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/h5py-2.6.0-np111py27_1/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/h5py-2.6.0-np111py27_1/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/iopro-1.7.2-np110py27_p0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/iopro-1.7.2-np110py27_p0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/iopro-1.7.2-np110py35_p0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/iopro-1.7.2-np110py35_p0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/matplotlib-1.5.1-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/matplotlib-1.5.1-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/mdp-3.5-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/mdp-3.5-py35_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/numba-0.25.0-np110py27_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/numexpr-2.5.2-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/numexpr-2.5.2-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/numexpr-2.5.2-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/numexpr-2.5.2-np111py27_0/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/numpy-1.10.4-py35_0/info/index.json:  "name": "numpy", 
/Users/ekruse/anaconda/pkgs/numpy-1.11.0-py27_0/info/index.json:  "name": "numpy", 
/Users/ekruse/anaconda/pkgs/odo-0.4.2-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/odo-0.4.2-py35_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/pandas-0.18.0-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/pandas-0.18.0-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/patsy-0.4.1-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/patsy-0.4.1-py35_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/pysal-1.11.1-py27_0/info/index.json:    "numpy", 
/Users/ekruse/anaconda/pkgs/pytables-3.2.2-np110py35_3/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/pytables-3.2.2-np110py35_3/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/pytables-3.2.2-np111py27_3/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/pytables-3.2.2-np111py27_3/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/scikit-image-0.12.3-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/scikit-image-0.12.3-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/scikit-image-0.12.3-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/scikit-image-0.12.3-np111py27_0/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/scikit-learn-0.17.1-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/scikit-learn-0.17.1-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/scikit-learn-0.17.1-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/scikit-learn-0.17.1-np111py27_0/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/scipy-0.17.0-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/scipy-0.17.0-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/scipy-0.17.0-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/scipy-0.17.0-np111py27_0/info/index.json:    "numpy 1.11", 
/Users/ekruse/anaconda/pkgs/statsmodels-0.6.1-np110py35_0/info/index.json:    "numpy 1.10*", 
/Users/ekruse/anaconda/pkgs/statsmodels-0.6.1-np110py35_0/info/index.json:    "numpy 1.10", 
/Users/ekruse/anaconda/pkgs/statsmodels-0.6.1-np111py27_0/info/index.json:    "numpy 1.11*", 
/Users/ekruse/anaconda/pkgs/statsmodels-0.6.1-np111py27_0/info/index.json:    "numpy 1.11", 
```

**Comment by mcg1969** [comment]
Can you post the entire output of `conda update --all`, including the upgrades?

**Comment by ethankruse** [comment]
Sure... there's not much there though.

```
ekruse@Ethan ~$ conda update --all
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ....
Solving package specifications: .........

Package plan for installation in environment /Users/ekruse/anaconda:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    freetype-2.4.10            |                1         655 KB
    libpng-1.5.13              |                1         149 KB
    qt-4.8.5                   |                3        36.2 MB
    hdf5-1.8.15.1              |                2         1.6 MB
    ipaddress-1.0.14           |           py27_0          27 KB
    numpy-1.9.3                |           py27_1         2.9 MB
    pillow-2.7.0               |           py27_0         453 KB
    pyparsing-2.0.3            |           py27_0          63 KB
    queuelib-1.2.2             |           py27_0          16 KB
    traits-4.4.0               |           py27_0         374 KB
    astropy-1.0.4              |       np19py27_0         5.3 MB
    h5py-2.5.0                 |       np19py27_3         708 KB
    iopro-1.7.2                |      np19py27_p0         584 KB
    numexpr-2.4.4              |       np19py27_0         103 KB
    pyface-4.4.0               |           py27_0         693 KB
    pyqt-4.10.4                |           py27_0         3.5 MB
    scipy-0.17.0               |       np19py27_0        12.9 MB
    matplotlib-1.4.3           |       np19py27_0        40.9 MB
    numba-0.25.0               |       np19py27_0         1.6 MB
    pandas-0.16.2              |       np19py27_0         5.4 MB
    pytables-3.2.2             |       np19py27_0         1.6 MB
    scikit-learn-0.16.1        |       np19py27_0         3.3 MB
    traitsui-4.4.0             |           py27_0         837 KB
    apptools-4.2.1             |           py27_0         282 KB
    basemap-1.0.7              |       np19py27_0       118.5 MB
    enable-4.4.1               |       np19py27_0         1.4 MB
    scikit-image-0.11.3        |       np19py27_0        16.1 MB
    statsmodels-0.6.1          |       np19py27_0         4.6 MB
    ------------------------------------------------------------
                                           Total:       260.7 MB

The following packages will be UPDATED:

    basemap:      1.0.7-np110py27_0  --> 1.0.7-np19py27_0 
    iopro:        1.7.2-np110py27_p0 --> 1.7.2-np19py27_p0
    numba:        0.25.0-np110py27_0 --> 0.25.0-np19py27_0
    scipy:        0.17.0-np111py27_0 --> 0.17.0-np19py27_0
    statsmodels:  0.6.1-np111py27_0  --> 0.6.1-np19py27_0 

The following packages will be DOWNGRADED:

    apptools:     4.3.0-py27_0       --> 4.2.1-py27_0     
    astropy:      1.1.2-np111py27_0  --> 1.0.4-np19py27_0 
    enable:       4.5.1-np110py27_0  --> 4.4.1-np19py27_0 
    freetype:     2.5.5-0            --> 2.4.10-1         
    geos:         3.4.2-0            --> 3.3.3-0          
    gevent:       1.1.1-py27_0       --> 1.1.0-py27_0     
    glueviz:      0.7.2-py27_0       --> 0.7.1-py27_0     
    h5py:         2.6.0-np111py27_1  --> 2.5.0-np19py27_3 
    hdf5:         1.8.16-0           --> 1.8.15.1-2       
    ipaddress:    1.0.16-py27_0      --> 1.0.14-py27_0    
    libpng:       1.6.17-0           --> 1.5.13-1         
    matplotlib:   1.5.1-np111py27_0  --> 1.4.3-np19py27_0 
    numexpr:      2.5.2-np111py27_0  --> 2.4.4-np19py27_0 
    numpy:        1.11.0-py27_0      --> 1.9.3-py27_1     
    pandas:       0.18.0-np111py27_0 --> 0.16.2-np19py27_0
    pillow:       3.2.0-py27_0       --> 2.7.0-py27_0     
    pyface:       4.5.2-py27_0       --> 4.4.0-py27_0     
    pyparsing:    2.1.1-py27_0       --> 2.0.3-py27_0     
    pyqt:         4.11.4-py27_1      --> 4.10.4-py27_0    
    pytables:     3.2.2-np111py27_3  --> 3.2.2-np19py27_0 
    qt:           4.8.7-1            --> 4.8.5-3          
    queuelib:     1.4.2-py27_0       --> 1.2.2-py27_0     
    scikit-image: 0.12.3-np111py27_0 --> 0.11.3-np19py27_0
    scikit-learn: 0.17.1-np111py27_0 --> 0.16.1-np19py27_0
    traits:       4.5.0-py27_0       --> 4.4.0-py27_0     
    traitsui:     4.5.1-py27_0       --> 4.4.0-py27_0     
    w3lib:        1.14.2-py27_0      --> 1.12.0-py27_0    

Proceed ([y]/n)? 
```

I'm using conda v 4.0.5 by the way.

**Comment by ethankruse** [comment]
So, for what it's worth, the problem has to do with the 'imaging' package. I uninstalled imaging and updates are proceeding as expected. Trying to reinstall imaging forces the downgrade back to numpy 1.9 and all the associated cascade of downgrades.

I can't quite figure out why.... imaging doesn't require numpy at all. It does specify freetype 2.4.10, so that downgrade makes sense, but that also doesn't have anything to do with numpy.

Meanwhile, imaging also supposedly requires zlib 1.2.7, but I have 1.2.8 and it's not trying to downgrade that one.

Anyway, all this to say that I uninstalled imaging and things are fine now, but I don't really get why that was causing problems.
